reference/supervisor.md

```diff
@@ -462,6 +462,24 @@ refer to [this section](#wb_supervisor_simulation_reset_physics).
 
 **Name**
 
+**wb\_supervisor\_node\_set\_visibility** - *set the visibility of a node*
+
+{[C++](cpp-api.md#cpp_node)}, {[Java](java-api.md#java_node)}, {[Python](python-api.md#python_node)}, {[Matlab](matlab-api.md#matlab_supervisor)}, {[ROS](ros-api.md)}
+
+```c
+#include <webots/supervisor.h>
+
+void wb_supervisor_node_set_visibility(WbNodeRef node, WbNodeRef from, bool visible);
+```
+
+**Description**
+
+The `wb_supervisor_node_set_visibility()` function sets the visibility of a node from the specified [Camera](camera.md), [Lidar](lidar.md), [RangeFinder](rangefinder.md) or [Viewpoint](viewpoint.md) node. An invisible node will not be rendered in the image or 3D scene. The `node` argument is the node to hide or show, the `from` argument must be either a [Camera](camera.md), [Lidar](lidar.md), [RangeFinder](rangefinder.md) or [Viewpoint](viewpoint.md) node. The `visible` argument specifies whether the node should be hidden (false) or shown (true). By default, all the nodes are visible. It is relevant to show a node only if it was previously hidden using this function.
```

Comment:
Now it is ok.

But I would personally rephrase it and integrate it with the rest of the description, for example:
[..] or [Viewpoint](viewpoint.md) node.
*In particular it defines if the node is visible in the image recorded by the `from` device.
The* `from` argument must be either a [Camera](camera.md), [Lidar](lidar.md), [RangeFinder](rangefinder.md) or [Viewpoint](viewpoint.md) node. *In case of the [Viewpoint](viewpoint.md) the node is hidden or shown in the main 3D scene.*
The `node` argument is the node to hide or show.
The `visible` argument specifies [...]

I have other questions about the the  `node`argument that I think should be answer in this description:
What kind of nodes can be specified in the `node`argument?
What happens if I specified for example a Solid or Group node? and a boundingObject node?